src/pybind/mgr/dashboard/tools.py

```diff
@@ -832,6 +832,7 @@ def getargspec(func):
             func = func.__wrapped__
     except AttributeError:
         pass
+    # pylint: disable=deprecated-method
```

Comment:
[How](https://github.com/pytest-dev/pluggy/pull/94/files) `pluggy` devs dealt with this...

Reply:
we're very likely going full Python 3 by the end of the year. I'd be ok with a pylint for the moment.